### PR TITLE
Add translation for 3.4dev6

### PR DESCRIPTION
**Pull request by cornelinux**
Working on #2342

### CODE REVIEW

File: po/nl.po [hunk 1]
```diff
@@ -819,6 +819,11 @@ msgstr "Maak een beleid"
 msgid "Create a new Event Handler"
 msgstr "Maak een nieuwe event handler"
 
+#: privacyidea/static/components/config/views/config.resolvers.http.html:5
+#, fuzzy
+msgid "Create a new HTTP Resolver"
+msgstr "Maak een nieuwe LDAP resolver"
```
Comment (plettich) [inline po/nl.po:825]: s/LDAP/HTTP/

File: po/nl.po [hunk 2]
```diff
@@ -3029,6 +3073,11 @@ msgstr "Takenmodule"
 msgid "Test Email sent successfully."
 msgstr "Test e-mail met succes verzonden."
 
+#: privacyidea/static/components/config/views/config.resolvers.http.html:86
+#, fuzzy
+msgid "Test HTTP Resolver"
+msgstr "Test LDAP-resolver"
```
Comment (plettich) [inline po/nl.po:3079]: s/LDAP/HTTP/

**Review by plettich — CHANGES_REQUESTED** [review]

File: po/nl.po (hunk 1 above)
Comment (cornelinux) [reply po/nl.po:825]: Your dutch is perfect! ;-)
I think these translations were "autoadded" by best match.
The NL translation was done *before* the HTTP resolver was merged.
So please ignore dutch. I will ask the translator from the community to update the durch translation.

File: po/nl.po (hunk 2 above)
Comment (cornelinux) [reply po/nl.po:3079]: s.o.

**Comment by cornelinux** [comment]
@plettich If you have no comments on the German translation, I would like to merge it.
The dutch translation follows by a dutchman

**Review by plettich — APPROVED** [review]
looks good, except for dutch ;-)

**Pull request merged**